server/planning/templates/assignment_mail_subject.txt

```diff
@@ -0,0 +1 @@
+Superdesk assignment planning: {{slugline}}
```

Comment:
I think it should be just `Superdesk assignment: {{slugline}}` no need to change it

Reply:
my bad